### DISCUSSION TITLE
Tres pantallas

### DISCUSSION THREAD

**Discussion by lucasditomase** [lucasditomase/Grupo-3#7] [Ideas]
Contexto: https://github.com/lucasditomase/Grupo-3/discussions/5

Progreso, Hábitos y Perfil

**Comment by tomisarapura** [comment]
Las barras de progrso pueden ir cambiando de color gradualmente a medida que se acercan a completar todos los habitos

**Reply by lucasditomase** [reply]
Menos de 50% ROJO
Más de 50% y menos de 70% AMARILLO
Más de 70% VERDE
